### PR TITLE
Feedback on documentation

### PR DESCRIPTION
**Pull request by JulienTant**
#### Summary
Creating this PR to address @wiggin77 remark on the new changes in the README ~ It's also a good chance for anyone to leave feedback on the documentation addition from https://github.com/mattermost/mattermost-plugin-nps/commit/b9e7b488abf1f6e860779c83ae930d25729f5139

#### Ticket Link
N/A OSF

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Before running `make deploy`, make sure you have set the following values in you
 
 ### How it works - overview
 
-The plugin send a survey after 45 days when a new (as in never seen before - including downgrade) version of Mattermost is detected. It also send a message to user 7 days after their registration to get early feedbacks, and allow users to give a feedback at any time they desire.
+The plugin sends a survey after 45 days when a new (as in never seen before - including downgrade) version of Mattermost is detected. It also sends a message to user 7 days after their registration to get early feedback, and allow users to give feedback at any time they desire.
 
 Survey and feedbacks are sent to rudder as `Track` events.
```
Comment (wiggin77) [inline README.md:35]:
```suggestion
Surveys and feedback are sent to rudder as `Track` events.
```

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Before running `make deploy`, make sure you have set the following values in you
 
 ### How it works - overview
 
-The plugin send a survey after 45 days when a new (as in never seen before - including downgrade) version of Mattermost is detected. It also send a message to user 7 days after their registration to get early feedbacks, and allow users to give a feedback at any time they desire.
+The plugin sends a survey after 45 days when a new (as in never seen before - including downgrade) version of Mattermost is detected. It also sends a message to user 7 days after their registration to get early feedback, and allow users to give feedback at any time they desire.
```
Comment (wiggin77) [inline README.md:33]:
```suggestion
The plugin sends a survey after 45 days when a new (as in never seen before - including downgrade) version of Mattermost is detected. It also sends a message to users 7 days after their registration to get early feedback, and allows users to give feedback at any time they desire.
```

**Review by pvev — APPROVED** [review]
thanks @JulienTant , LGTM, besides already given feedback you can 🚢

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Review by julmondragon — APPROVED** [review]
Thanks @JulienTant !

**Pull request merged**